azurerm/internal/services/apimanagement/api_management_policy_resource.go

```diff
@@ -0,0 +1,202 @@
+package apimanagement
+
+import (
+	"fmt"
+	"html"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/apimanagement/mgmt/2019-12-01/apimanagement"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/apimanagement/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmApiManagementPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmApiManagementPolicyCreateUpdate,
+		Read:   resourceArmApiManagementPolicyRead,
+		Update: resourceArmApiManagementPolicyCreateUpdate,
+		Delete: resourceArmApiManagementPolicyDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Read:   schema.DefaultTimeout(5 * time.Minute),
+			Update: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"api_management_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: azure.ValidateResourceID,
+			},
+
+			"xml_content": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Computed:         true,
+				ConflictsWith:    []string{"xml_link"},
+				DiffSuppressFunc: XmlWithDotNetInterpolationsDiffSuppress,
+			},
+
+			"xml_link": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: []string{"xml_content"},
+			},
```

Comment:
Should these have `ExactlyOneOf` (or `AtLeastOneOf`) set?

Reply:
I wasn't aware of that option, but yes, that would be the right thing to do.

Reply:
Done in 69bf29a7e9f7e6f06b3a8a21132fbd68caee9c42.